content/en/docs/pipelines/installation/overview.md

```diff
@@ -4,41 +4,37 @@ description = "Overview of the ways to deploy Kubeflow Pipelines"
 weight = 10
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
 Kubeflow Pipelines offers a few installation options.
 This page describes the options and the features available
 with each option:
 
-* A [standalone](#standalone) Kubeflow Pipelines deployment is the minimal
+* [Kubeflow Pipelines Standalone](#kubeflow-pipelines-standalone) is the minimal
 portable installation that only includes Kubeflow Pipelines.
-* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow) provides
+* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow-deployment) provides
 all Kubeflow components and more integration with each platform.
-* **Beta**: [Google Cloud AI Platform Pipelines](#marketplace) provides easy-to-use management
-UI for installing and using Kubeflow Pipelines on Google Cloud.
+* **Beta**: [Google Cloud AI Platform Pipelines](#google-cloud-ai-platform-pipelines) provides easy-to-use management
+UI for installing and using Kubeflow Pipelines Standalone on Google Cloud.
 
 ## Choosing an installation option
 
 1. Do you want to use other Kubeflow components in addition to Pipelines?
 
-    If yes, choose the [full Kubeflow](#full-kubeflow).
+    If yes, choose the [full Kubeflow](#full-kubeflow-deployment).
```

Comment:
If yes, choose the [full Kubeflow](#full-kubeflow-deployment).

->

If yes, choose the [full Kubeflow](#full-kubeflow-deployment) deployment.